draft-ietf-dprive-dnsoquic.md

```diff
@@ -536,11 +537,11 @@ fall back to DoT and then potentially clear text, as specified in DoT
 {{!RFC8310}}, depending on their privacy profile.
 
 DNS clients SHOULD remember server IP addresses that don't support DoQ.
-Timeouts, connection refusals, and QUIC handshake failures are valid indicators
```

Comment:
They are subject to spoofing by attacker, and so not fully reliable/valid, but they remain the best indicators we have available.

Reply:
OK.